lodash-plugin/src/test/java/com/github/underscore/lodash/StringTest.java

```diff
@@ -1518,53 +1518,53 @@ public void chain() {
         $.chain(new String[] {""}).rest();
         $.chain(new String[] {""}).rest(1);
         $.chain(new String[] {""}).flatten();
-        $.chain(new Integer[] {0}).map(new Function1<Integer, Integer>() {
+        $.chain(new Integer[] {0}).map(new Function<Integer, Integer>() {
```

Comment:
\[FindBugs\] INFO: RV: Return value of $$Chain\.map\(Function\) ignored, is this OK in com\.github\.underscore\.lodash\.StringTest\.chain\(\)